### ISSUE TITLE
Requirements for submitting a home

### ISSUE THREAD

**Issue by davisgibson** [davisgibson/davisgibson#13]
- Full Address
- Price?
- Description

Does it need detailed information such as washer/dryer, sqft or beds/baths?

**Comment by jordanx9** [comment]
Yeah, they include sq ft. Optional if they would like to include Washer/Dryer Bc this could be a negotiable factor

**Comment by jordanx9** [comment]
Beds & Bath also included and required
